### PR TITLE
Security and permission

### LINKED ISSUE #11: [Feature] Restrict printing to specific group(s)
Hi and thanks for your app!

Is it possible to add an option that defines Nextcloud groups allowed to print?

### PR DESCRIPTION
**Pull request by simmstein**
- Improve security (escape command line args using symfony/process)
- Add option to define groups allowed to print (fix #11)

You must run `composer install` to install dependencies (php >= 7.0 required).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "symfony/process": "^4.4"
```
Comment (kesselb) [inline composer.json]: Bad idea as Nextcloud already has `symfony/process` as peer dependency: https://github.com/nextcloud/3rdparty/tree/master/symfony

**Comment by e-alfred** [comment]
Thanks for your contribution! Are you interested in collaborating on this app in the future to make it feature complete?

**Comment by simmstein** [comment]
Yeap!

**Comment by simmstein** [comment]
Do you want to merge this PR?

**Comment by e-alfred** [comment]
@simmstein I merged your PR and invited you as a collaborator. Thanks for your help to improve this app!

**Pull request merged**